server/test/debian_10_event_log.yml

```diff
@@ -0,0 +1,164 @@
+// Fetched via `tpm2_eventlog debian_10_binary_bios_measurements`
+// Note this is invalid yaml.
+---
+events:
```

Comment:
We should use output from a Newer GCP VM to at least get the EV_NONHOST_INFO stuff.

Reply:
Done.